entries/t/twingate.com.json

```diff
@@ -0,0 +1,17 @@
+{
+  "Twingate": {
+    "domain": "twingate.com",
+    "tfa": [
+      "totp",
+      "custom-hardware"
+    ],
+    "custom-hardware": [
+      "Biometrics",
+      "Yubico OTP"
+    ],
+    "documentation": "https://www.twingate.com/docs/two-factor-authentication-security-policies",
+    "categories": [
+      "remote"
```

Comment:
I think security may be a more appropriate category. Or perhaps both
```suggestion
      "security"
```

Reply:
I feel a little confused about the classification here, but I personally prefer remote. ❤️ @Carlgo11

Reply:
Okay! Let's do remote then.